### PR TITLE
project-mongo-api Elle Frankenberg

### PR DESCRIPTION
**Pull request by ElleFrankenberg**
https://goldenglobe-express-api.herokuapp.com/nominees

### CODE REVIEW

**Review by sirimatsdotter — APPROVED** [review]
Great work Elle! I believe you nailed all the blue requirements and all end points work when I test them. Nice use of regex in the queries and overall clean and neat code! :)

File: server.js [hunk 1]
```diff
@@ -1,39 +1,95 @@
...
-//   PORT=9000 npm start
+const nomineeSchema = new mongoose.Schema({
+  year_film: Number,
+  year_award: Number,
+  ceremony: Number,
+  category: String,
+  nominee: String,
+  film: String,
+  win: Boolean
+})
```
Comment (KarolCislowski) [inline server.js:22]: nice schema 👍🏻

File: server.js [hunk 2]
```diff
@@ -1,39 +1,95 @@
...
+      }
+    })
+    res.json(nominees)
+  } else {
+    const nominees = await Nominee.find()
+    res.json(nominees)
+  }  
+  } catch (error) {
+    res.status(404).send({ error: "Not found" })
+  }
```
Comment (KarolCislowski) [inline server.js:70]: nice filtering

File: server.js [hunk 3]
```diff
@@ -1,39 +1,95 @@
...
+
+  try {
+    const nominee = await Nominee.findById(nomineeId)
+    if (nominee) {
+      res.json(nominee)
+    } else {
+      res.status(404).json({ error: 'Not found' })
+    }
+  } catch (error) {
+    res.status(400).json({ error: 'Invalid ID' })
```
Comment (KarolCislowski) [inline server.js:85]: nice choice of error statuses 👍🏻

**Review by KarolCislowski — APPROVED** [review]
Great API 👍🏻

**Pull request closed without merging**